src/diagram/components/quantity-node.tsx

```diff
@@ -176,6 +176,7 @@ const _QuantityNode: React.FC<IProps> = ({ data, isConnectable }) => {
         <div className={classNames("variable-info-row", "description-row", { expanded: showDescription })}>
           {showDescription && 
             <TextareaAutosize
+              autoComplete="off"
```

Comment:
Another instance of the Notes fields for which we do want to allow spellcheck. So we only disable autocomplete here.